### ISSUE TITLE
Migrate the db seeding functionality of devtools to an npm package

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction-devtools#18]
We should abstract the database-seeding functionality out of dev-tools and into an npm package, and then depend on and use that npm package from this Reaction plugin.

**Comment by zenweasel** [comment]
We have already created an npm package that does this.  https://github.com/reactioncommerce/reaction-data-loader

**Comment by spencern** [comment]
@zenweasel that repo is private, could the database seeding functionality of that be 
1. made public?
2. Integrated as the solution for loading data via devtools?

**Comment by zenweasel** [comment]
Yeah, we're working on that. Hope to have it done soon. We will publish data-loader as an NPM package that devtools can just depend on. Should have mentioned that in my earlier comment.
